theano/sandbox/cuda/basic_ops.py

```diff
@@ -3035,8 +3038,7 @@ def c_code_cache_version(self):
     def c_code(self, node, name, inputs, outputs, sub):
```

Comment:
As the c code was changed, we normally raise the value returned by c_code_cache_version. Can you do that? In the current state, it is to not do this, but it bring some inconsistency in the cache and I would prefer to not have this.